.storybook/stories/CorporateCampaign.stories.js

```diff
@@ -9,6 +9,7 @@ import CorporateCampaignLanding from '@/pages/LandingPages/CorporateCampaign/CCL
 import CampaignHero from '@/components/CorporateCampaign/CampaignHero';
 import CampaignStatus from '@/components/CorporateCampaign/CampaignStatus';
 import cookieStoreStoryMixin from '../mixins/cookie-store-story-mixin';
+import DynamicHeroClassic from '@/components/Contentful/DynamicHeroClassic';
```

Comment:
Can you please create a new Story for DynamicHeroClassic so we don't conflate it with Corporate Campaigns.

Reply:
Otherwise this is looking good!